string    ROBOT_CLASS=robot
string   OBJECT_CLASS=object
string LOCATION_CLASS=location
string mdp_class
string name
string x
string y
string orientation
string last_x
string last_y
string last_orientation
